### PR TITLE
cors fix try2

### PR DESCRIPTION
**Pull request by MrMarcelM**
cors enable fix attempt 2

### CODE REVIEW

File: Know-Your-Nation-Speedy/Know-Your-Nation-Speedy/Startup.cs [hunk 1]
```diff
@@ -32,25 +39,28 @@ public void ConfigureServices(IServiceCollection services)
 
             var connection = @"Server=dev.retrotest.co.za;Database=ereader;User Id=group4;Password=3bHNuE8&rvG+99U2;";
```
Comment (ash-rabbit) [inline Know-Your-Nation-Speedy/Know-Your-Nation-Speedy/Startup.cs]: Should there be space between *User Id* = group4?

Comment (ash-rabbit) [inline Know-Your-Nation-Speedy/Know-Your-Nation-Speedy/Startup.cs:40]: Should there be space between *User Id* = group4?

File: Know-Your-Nation-Speedy/Know-Your-Nation-Speedy/Startup.cs [hunk 2]
```diff
@@ -32,25 +39,28 @@ public void ConfigureServices(IServiceCollection services)
...
             services.AddSingleton(Configuration);
         }
 
         // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
         public void Configure(IApplicationBuilder app, IHostingEnvironment env)
         {
-            // Enable middleware to serve generated Swagger as a JSON endpoint.
-            app.UseSwagger();
+            app.UseCors("CorsPolicy");
+            // UpdateDatabase(app);
```
Comment (FredJEhlers) [inline Know-Your-Nation-Speedy/Know-Your-Nation-Speedy/Startup.cs]: I can't see a problem apart from this Ghost Code

File: Know-Your-Nation-Speedy/Know-Your-Nation-Speedy/Startup.cs [hunk 3]
```diff
@@ -32,25 +39,28 @@ public void ConfigureServices(IServiceCollection services)
 
             var connection = @"Server=dev.retrotest.co.za;Database=ereader;User Id=group4;Password=3bHNuE8&rvG+99U2;";
             services.AddDbContext<MyDbContext>
-                (options => options.UseSqlServer(connection));
+                (options => options.UseSqlServer(connection)); //other option: UseSqlServer(Configuration.GetSection(connection))
```
Comment (ash-rabbit) [inline Know-Your-Nation-Speedy/Know-Your-Nation-Speedy/Startup.cs]: So this commented out code (GC) should then be something like
*//  TODO: other option: 
 //  UseSqlServer(Configuration.GetSection(connection))*
If you really wanna keep it, same applies for line 50

File: Know-Your-Nation-Speedy/Know-Your-Nation-Speedy/Startup.cs (hunk 2 above)
Comment (MrMarcelM) [reply Know-Your-Nation-Speedy/Know-Your-Nation-Speedy/Startup.cs]: Thank you Fred. I will remove the ghost code. Were primarily just trying to get the cors problem solved.

**Review by ash-rabbit — CHANGES_REQUESTED** [review]
I've added 2 comments

File: Know-Your-Nation-Speedy/Know-Your-Nation-Speedy/Startup.cs (hunk 1 above)
Comment (MrMarcelM) [reply Know-Your-Nation-Speedy/Know-Your-Nation-Speedy/Startup.cs:40]: Hi ash, the connection string has been tested and it is currently working. (I'm on front end generally so just picking up from where they left off) as far as I understand apparently yes?

File: Know-Your-Nation-Speedy/Know-Your-Nation-Speedy/Startup.cs (hunk 3 above)
Comment (MrMarcelM) [reply Know-Your-Nation-Speedy/Know-Your-Nation-Speedy/Startup.cs]: thank you I've incorporated your advice.

File: Know-Your-Nation-Speedy/Know-Your-Nation-Speedy/Startup.cs (hunk 1 above)
Comment (ash-rabbit) [reply Know-Your-Nation-Speedy/Know-Your-Nation-Speedy/Startup.cs:40]: I'm happy

File: Know-Your-Nation-Speedy/Know-Your-Nation-Speedy/Startup.cs (hunk 3 above)
Comment (ash-rabbit) [reply Know-Your-Nation-Speedy/Know-Your-Nation-Speedy/Startup.cs]: You're welcome, once its changed I'll approve :)

**Review by FredJEhlers — APPROVED** [review]

**Review by ash-rabbit — APPROVED** [review]
Thanks Marcel for being the first one to resolve my comments the right way

**Pull request merged**